src/App.js

```diff
@@ -1,21 +1,17 @@
 import React, { Component } from "react";
-import logo from "./logo.svg";
 import "./App.scss";
-import {BrowserRouter, Route, Switch, NavLink } from "react-router-dom";
+import {BrowserRouter, Route, Switch } from "react-router-dom";
 import "./App.sass";
-import Dummy from "./js/components/Dummy";
 import Home from "./js/components/Home/Home";
-import Navbar from "./js/components/Navbar/Navbar";
-import { Provider } from "react-redux";
 import Books from "./js/components/Books/Books";
 import Login from "./js/components/LoginPage/Login";
 import SignUp from "./js/components/RegisterPage/SignUp";
 import ForgotSignIn from "./js/components/ForgotSignIn/ForgotSignIn";
 
 class App extends Component {
   render() {
-    var loc_navBarTitle = "KnowYourNation";
-    var loc_navbarItems = [false, true, true, true];
+    // var loc_navBarTitle = "KnowYourNation";
```

Comment:
Make use of let when declaring your variables.

Stick to a specific naming convention for your variables.
If you and your team decide to use underscore throughout your project, use it.
If you and your team decide to use camel case, use it.
To make it easier, you can use underscore for constants and camel case for declarations and capital in the beginning for your constructors.
Avoid mixing different styles in one project.